lib/msal-angular/src/msal.service.ts

```diff
@@ -133,7 +133,7 @@ export class MsalService extends UserAgentApplication {
                     // id_token or access_token can be renewed
                     if (window.parent === window && !window.parent.callBackMappedToRenewStates[requestInfo.stateResponse]) {
                         if (token) {
-                            this.broadcastService.broadcast("msal:acquireTokenSuccess", token);
+                            this.broadcastService.broadcast("msal:acquireTokenSuccess", {token});
```

Comment:
In case of success_events, some places broadcast both token and token_type and some places broadcast just token. Can we please add token_type to all events as well. Also the reason to pass objects in broadcast events was to enable us to pass scopes to the developer which they can pass in subsequent AT calls.

Reply:
Sure. I just noticed that this is missing in msal-angularjs as well. Shall we fix it everywhere?
https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-angularjs/lib/msal-angular.js#L452